tests/integration/smoke_test.go

```diff
@@ -98,6 +99,10 @@ func (suite *SmokeSuite) TestRookClusterInstallation_smokeTest() {
 }
 
 func (suite *SmokeSuite) TestOperatorGetFlexvolumePath() {
+	v := version.MustParseSemantic(suite.k8sh.GetK8sServerVersion())
```

Comment:
skipping my beautiful test? :(

perhaps link to our Rook issue as well? https://github.com/rook/rook/issues/1330

Reply:
updated the skip message, linked rook issue #1330, which has the k8s issue in the comments.